python/tvm/relax/op/base.py

```diff
@@ -51,12 +51,12 @@ def call_tir(
     tir_vars: Optional[Union[ShapeExpr, Tuple[PrimExpr], List[PrimExpr]]] = None,
 ) -> Call:
     """
-    Call a destination-passing-style function and return the output.
+    Call a tir.prim_func function and return the output.
 
     Parameters
     ----------
     func : Union[str, Expr]
```

Comment:
I think the signature in Python should reflect that we expect the argument to be a global var if that's how we're going to implement it.